src/controllers/OfficeControllers.js

```diff
@@ -0,0 +1,79 @@
+import db from '../models/index';
+import Helper from '../helpers/Helper';
+
+class OfficeController {
+  static async createOffice(req, res) {
+    const createQuery = `INSERT INTO
+    offices(type, name, createdBy, createdOn)
+    VALUES($1, $2, $3, $4)
+    returning *`;
+
+    const values = [
+      req.body.type,
+      req.body.name,
+      req.user.id,
+      req.body.createdOn,
+    ];
+ 
+    try {
+      const { rows } = await db.query(createQuery, values);
+      return res.status(201).send({
+        status: 201,
+        data: [{
+          id: rows[0].id,
+          message: 'Political office created',
```

Comment:
Same here the challenge spec says you should include office name and type

Reply:
done